server/pulp/server/controllers/distributor.py

```diff
@@ -237,6 +237,8 @@ def update(repo_id, dist_id, config=None, delta=None):
 
     if not valid_config:
         raise exceptions.PulpDataException(message)
+
+    distributor.last_updated = dateutils.now_utc_datetime_with_tzinfo()
```

Comment:
How is this being saved? The distributor object doesn't have a last_updated field. I see it's model definition here:

https://github.com/pulp/pulp/blob/51b36d4a5e19e68308f02adfbce1c4bf08144b5f/server/pulp/server/db/model/__init__.py#L1211-L1221